components/description_list/description_list.vue

```diff
@@ -0,0 +1,122 @@
+<!-- eslint-disable vue/require-v-for-key -->
+<template>
+  <dl :class="['dt-description-list', getDirectionClass, getGapClass]">
+    <template
+      v-for="item in items"
+    >
+      <dt
+        :class="['dt-description-list__term', termClass]"
+      >
+        {{ item.term }}
+      </dt>
+      <dd
+        :class="['dt-description-list__description', descriptionClass]"
```

Comment:
Would update to sth like a computed propery `ddClass` and `dtClass`

```
dtClass() {
    return ['dt-description-list__term', this.termClass];
  }
```

Result in: `<dt :class="dtClass">`